src/posix/platform/radio.cpp

```diff
@@ -978,6 +978,18 @@ otError otPlatRadioReceiveAt(otInstance *aInstance, uint8_t aChannel, uint32_t a
     return OT_ERROR_NOT_IMPLEMENTED;
 }
 
+#if OPENTHREAD_CONFIG_PLATFORM_BOOTLOADER_MODE_ENABLE
+otError otPlatResetToBootloader(otInstance *aInstance)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+    otError error = OT_ERROR_NONE;
+
+    error = GetRadioSpinel().SendReset(SPINEL_RESET_BOOTLOADER);
+
+    return error;
```

Comment:
I think the code can be simplified to `return GetRadioSpinel().SendReset(SPINEL_RESET_BOOTLOADER);`